lib/rubocop/config.rb

```diff
@@ -567,6 +573,33 @@ def target_ruby_version_from_version_file
         end
     end
 
+    def target_rails_version_from_bundler_lock_file
+      @target_rails_version_from_bundler_lock_file ||=
+        read_rails_version_from_bundler_lock_file
+    end
+
+    def read_rails_version_from_bundler_lock_file
+      lock_file_path = bundler_lock_file_path
```

Comment:
AFAIK, If this cop search strictly for lockfile, it need to use the following Bundler's method.

`Bundler::SharedHelpers.default_lockfile`

Because the mechanism to search the lockfile is complicated as follows.
https://github.com/bundler/bundler/blob/v1.16.1/lib/bundler/shared_helpers.rb#L240-L246

Of course, now RuboCop doesn't depend on Bundler, so we can not use `Bundler::SharedHelpers.default_lockfile`. Please refer to when considering @pocke's advice 😃 
https://github.com/bbatsov/rubocop/issues/5473#issuecomment-359242125

Reply:
Not a big concern right now.